frontend/packages/console-telemetry-plugin/src/listeners/const.ts

```diff
@@ -1,3 +1,10 @@
-export const SEGMENT_API_KEY = window.SERVER_FLAGS?.telemetry?.SEGMENT_API_KEY ?? '';
-export const TELEMETRY_DISABLED = window.SERVER_FLAGS?.telemetry?.DISABLED === 'true';
+export const SEGMENT_API_KEY =
+  window.SERVER_FLAGS?.telemetry?.DEVSANDBOX_SEGMENT_API_KEY ||
+  window.SERVER_FLAGS?.telemetry?.SEGMENT_API_KEY ||
```

Comment:
With this we are giving priority to devsandbox-segment-key but anyways we should not have more than one key at a single time so this should be ok.

Reply:
In fact the idea is the priorities the devsandbox key here, because the SRE team sets one that is not changeable from the devsandbox team. This is to override it. I hope that we find a better way, but it's to have a plan b.